suites/pre-release/http302/googleCredentialsTest.js

```diff
@@ -0,0 +1,70 @@
+Feature('Google Credentials - BDcat testing for release sign off');
+
+// To be executed with GEN3_SKIP_PROJ_SETUP=true
+// No need to set up program / retrieve access token, etc.
+const { expect } = require('chai');
+const fenceProps = require('../../../services/apis/fence/fenceProps.js');
+const { interactive, ifInteractive } = require('../../../utils/interactive.js');
+const {
+  getAccessTokenHeader,
+  requestUserInput,
+} = require('../../../utils/apiUtil');
+
+// Test elaborated for DataSTAGE but it can be reused in other projects
+const TARGET_ENVIRONMENT = process.env.GEN3_COMMONS_HOSTNAME || 'qa-dcp.planx-pla.net';
+
+BeforeSuite(async ({ I }) => {
+  console.log('Setting up dependencies...');
+  I.cache = {};
+
+  if (process.env.TEST_CLIENT_ID === undefined) {
+    throw new Error(`ERROR: There is no client_id defined for this ${TARGET_ENVIRONMENT} Test User. Please declare the "TEST_CLIENT_ID" environment variable and try again. Aborting test...`);
+  } else if (process.env.TEST_SECRET_ID === undefined) {
+    throw new Error(`ERROR: There is no secret_id defined for this ${TARGET_ENVIRONMENT} Test User. Please declare the "TEST_SECRET_ID" environment variable and try again. Aborting test...`);
+  } else if (process.env.TEST_IMPLICIT_ID === undefined) {
+    throw new Error(`ERROR: There is no implicit_id defined for this ${TARGET_ENVIRONMENT} Test User. Please declare the "TEST_IMPLICIT_ID" environment variable and try again. Aborting test...`);
+  }
+
+  I.TARGET_ENVIRONMENT = TARGET_ENVIRONMENT;
+  // Fetching public list of DIDs
+  const httpResp = await I.sendGetRequest(
+    `https://${TARGET_ENVIRONMENT}/index/index`,
+  );
+
+  I.cache.records = httpResp.data.records;
```

Comment:
the test doesnt need ClientID/ ClientSecret and also the indexd records, you can remove this part from before block. It will only make the test execution slow

Reply:
OK